spec/features/welcome/index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'welcome index' do
+  before :each do
+    visit root_path
+  end
+
+  describe 'form' do
+    it 'has a form' do
+      expect(current_path).to eq(root_path)
+      expect(page).to have_content("Ingredient Search")
+    end
+  end
+end
```

Comment:
Seems like you may be having some trouble with testing. I think that testing would be a great focus area for you going forward. Below is a little 'cheat sheet' on what you can think about when testing services, poros, facades: 

For testing, once you have it all refactored like you do now, I like to think of the following: 

PORO test: 
- tests any attributes as well as any methods (you didn't have any) that are in the poro)
- write a test that takes in the data that you're assuming you'll pass to it (json object with food data)
- test that when you call methods (attributes) on that object, that you get what you'd expect back 

Facade test: 
- write a test that tests the return value of the facade -- in your case, you want your facade method to return food objects, so test that when the method gets called, that the objects you get returned are all food objects 

Service test: 
- this one feels a bit vague, but you really just need to test the structure you're expecting to get back 
- usually I'll hit the endpoint using postman, and take a look at that response to get an idea for what I can test 
- you'll test the structure (key's and values) of the json response you're expecting the service call to return to you.